### PR TITLE
stepper should be draggable

### PR DESCRIPTION
**Pull request by VKobeliatsky**
Draggable component should allow the user  to start dragging the mouse over it in up/down directions. This should call onUp/onDown respectively on mouse moves. We would want this to be used in NumberInput, so that user can control its values with Stepper drags.

### CODE REVIEW

File: src/components/stepper/stepper.tsx [hunk 1]
```diff
@@ -8,38 +9,110 @@ import styles from './stepper.st.css';
...
+        clientY: 0
+    };
+
+    public render() {
+        const {dragged} = this.state;
+        const {
+            onUp,
+            onDown,
+            disableUp = false,
+            disableDown = false,
```
Comment (tyv) [inline src/components/stepper/stepper.tsx]: why not defaultProps?

Comment (VKobeliatsky) [inline src/components/stepper/stepper.tsx]: Remainder from when it was a stateless component. Will fix, thanks.

**Comment by VKobeliatsky** [comment]
@wtfil
please, take a look

**Comment by VKobeliatsky** [comment]
addresses some of #254

File: src/components/stepper/stepper.tsx [hunk 2]
```diff
@@ -1,47 +1,139 @@
...
+
+    private handlerClickUp: React.MouseEventHandler<HTMLButtonElement> =
+        ({altKey, ctrlKey, shiftKey}) => this.props.onUp({altKey, ctrlKey, shiftKey})
+
+    private handlerClickDown: React.MouseEventHandler<HTMLButtonElement> =
+        ({altKey, ctrlKey, shiftKey}) => this.props.onDown({altKey, ctrlKey, shiftKey})
+
+    private handleDragStart: React.MouseEventHandler<HTMLDivElement> =
+        ({clientX, clientY}) => {
+            this.setState({dragged: true});
```
Comment (wtfil) [inline src/components/stepper/stepper.tsx]: `dragged` better to keep in instance `dragRefPoint`. No needs to render when this flag is changed

**Pull request merged**